internal/configs/version2/template_helper.go

```diff
@@ -129,6 +129,7 @@ var helperFunctions = template.FuncMap{
 	"hasSuffix":         strings.HasSuffix,
 	"toLower":           strings.ToLower,
 	"toUpper":           strings.ToUpper,
+	"replaceAll":        strings.ReplaceAll,
```

Comment:
can we have a separate PR for this using a branch instead of `main`? We then can also include a reference to https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/custom-annotations/#helper-functions in all required docs and examples

Reply:
Created PR https://github.com/nginxinc/kubernetes-ingress/pull/5468